mlflow/pyfunc/__init__.py

```diff
@@ -557,7 +557,10 @@ def _load_model_or_server(model_uri: str, env_manager: str):
         env_root_dir=env_root_dir,
     )
     _logger.info("Restoring model environment. This can take a few minutes.")
-    pyfunc_backend.prepare_env(model_uri=local_path, capture_output=False)
+    # Set capture_output to True in Databricks so that when environment preparation fails, the
+    # exception message of the notebook cell output will include child process command execution
+    # stdout/stderr output.
+    pyfunc_backend.prepare_env(model_uri=local_path, capture_output=is_in_databricks_runtime())
```

Comment:
For this I'm just mirror the spark_udf approach here
https://github.com/mlflow/mlflow/blob/277e88bce6e3613b1ea006ec095cba569c45a5ae/mlflow/pyfunc/__init__.py#L916-L931